builtin/credential/okta/path_config.go

```diff
@@ -0,0 +1,148 @@
+package okta
+
+import (
+	"fmt"
+	"github.com/fatih/structs"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+	"github.com/sstarcher/go-okta"
+	"net/url"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `config`,
+		Fields: map[string]*framework.FieldSchema{
+			"organization": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Okta organization to authenticate against",
+			},
+			"token": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Okta admin API token",
+			},
+			"base_url": &framework.FieldSchema{
+				Type: framework.TypeString,
+				Description: `The API endpoint to use. Useful if you
+are using Okta development accounts.`,
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.ReadOperation:   b.pathConfigRead,
+			logical.UpdateOperation: b.pathConfigWrite,
+		},
+
+		HelpSynopsis: pathConfigHelp,
+	}
+}
+
+// Config returns the configuration for this backend.
+func (b *backend) Config(s logical.Storage) (*ConfigEntry, error) {
+	entry, err := s.Get("config")
+	if err != nil {
+		return nil, err
+	}
+
+	var result ConfigEntry
+	if entry != nil {
+		if err := entry.DecodeJSON(&result); err != nil {
+			return nil, err
+		}
+	}
+
+	return &result, nil
+}
+
+func (b *backend) pathConfigRead(
+	req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
+
+	cfg, err := b.Config(req.Storage)
+	if err != nil {
+		return nil, err
+	}
+	if cfg == nil {
+		return nil, nil
+	}
+
+	resp := &logical.Response{
+		Data: structs.New(cfg).Map(),
+	}
+
+	return resp, nil
+}
+
+func (b *backend) pathConfigWrite(
```

Comment:
The latest commit should add support for Create/Update - https://github.com/hashicorp/vault/pull/1966/commits/95d10523a2834e60e4481c357fe52cdc1d973e0a